workspaces/diff-engine/src/events/spec_chunk.rs

```diff
@@ -0,0 +1,332 @@
+use super::{RfcEvent, SpecEvent};
+use cqrs_core::Event;
+use serde_json;
+use std::convert::TryFrom;
+
+#[derive(Debug)]
+pub enum SpecChunkEvent {
+  Root(RootChunkEvent),
+  Batch(BatchChunkEvent),
+  Unknown(UnknownChunkEvent),
+}
```

Comment:
The `Root` chunk is what represent the `specification.json` and is considered to be in-order and consistent with itself. The `Batch` chunk is what we consider the new spec change files will read as (more info on the logic below). `Unknown` is to make sure we have a way to detect where any new spec change files don't meet the required standard, and ignored in further assembly of the spec events.